spec/factories/services.rb

```diff
@@ -17,15 +17,13 @@
             :repository,
             name: "#{name_prefix}-Installer-Updates",
             mirroring_enabled: evaluator.mirroring_enabled,
-            installer_updates: true,
-            enabled: false
+            installer_updates: true
```

Comment:
Why is this change needed? Background: `installer_updates` repos are usually set to disabled.

Reply:
@digitaltom Every time this factory called, it creates 4 repos, but two of these repos are always `enabled: false`.
```ruby
let(:product) { create :product , : with_mirrored_repositories }
$ product.repositories.pluck(:name, :scc_id, :enabled, :mirroring_enabled, :last_mirrrored_at)
=> [["Product 80-Pool", 271, true, true, nil],
 ["Product 80-Updates", 272, true, true, nil],
 ["Product 80-Installer-Updates", 273, false, true, nil],
 ["Product 80-Debuginfo-Updates", 274, false, true, nil]]
```
Using this factory along with `products` factory, we can create four different variations of the product with repositories (`:with_mirrored_repositories`, `:with_disabled_mirrored_repositories`, `:with_disabled_not_mirrored_repositories`, `:with_not_mirrored_repositories`). So every time we creating a product with mirrored repositories (enabled and mirrored), there's always two disabled repos. Even though usually `installer_updates` are set to disabled, I suggest this change just to cover more possible cases as it gives us the flexibility of creating more product variants.

Reply:
> it creates 4 repos, but two of these repos are always `enabled: false`.

That replicates a normal product setup -- two mandatory repos (Pool, Updates) and two non-mandatory repos (DebugInfo, Source).

(`repository.enabled` means enabled on the client side by default, as in `zypper mr -e`)

Reply:
Hi @ikapelyukhin ,

Okay, but may I suggest that the goal is to test every possible variation, not to replicate the default product setup? Also, removing this value doesn't seem to break any tests.

Reply:
Real product setup is a reasonable default, it doesn't interfere with testing all possible combinations in any way.
In principle, I don't object to changing this; however I can't guarantee that there are no side effects with this change either, so I personally wouldn't change it.
But the decision of whether to change it or not is really up to @digitaltom and other RMT maintainers :grimacing:

Reply:
I would keep it as it was before (with `enabled: false` on DebugInfo and Installer repos) to keep the default factory  data close to real-world. 
The four testcases (`:with_mirrored_repositories, :with_disabled_mirrored_repositories, :with_disabled_not_mirrored_repositories, :with_not_mirrored_repositories`) might even get merged into just 2, because there is always a mix of enabled + disabled repos in the output. I don't see the benefit in extra testing of a product with *only* enabled/disabled repos.

Reply:
@digitaltom @ikapelyukhin Alright, I'll leave it as it was before.